src/Kinds/K8sCronJob.php

```diff
@@ -38,7 +40,7 @@ class K8sCronJob extends K8sResource implements InteractsWithK8sCluster, Watchab
     /**
      * Set the job template.
      *
-     * @param  array|\RenokiCo\PhpK8s\Kinds\K8sJob  $job
```

Comment:
This style got highlighted in PhpStorm and I gave in a little to let it extract those in files I touched. If there's a strong preference one way or another or a way to encourage IDEs to favor a style, that could be nice.